packages/volto/.eslintrc

```diff
@@ -24,33 +25,50 @@
     "no-debugger": 1,
     "prettier/prettier": [
       "error",
-      { "trailingComma": "all", "singleQuote": true },
+      { "trailingComma": "all", "singleQuote": true }
     ],
     "react-hooks/rules-of-hooks": "error",
     "react-hooks/exhaustive-deps": "warn",
     "react/react-in-jsx-scope": "off",
-    "jsx-a11y/label-has-associated-control": "off",
+    "jsx-a11y/label-has-associated-control": "off"
   },
   "settings": {
+    "import/named": "error",
+    "react/jsx-filename-extension": [1, { "extensions": [".tsx", ".jsx"] }],
+    "no-restricted-imports": [
+      "error",
+      {
+        "name": "@plone/volto/components",
+        "message": "Importing from barrel files is not allowed. Please use direct imports of the modules instead."
+      },
+      {
+        "name": "@plone/volto/helpers",
+        "message": "Importing from barrel files is not allowed. Please use direct imports of the modules instead."
+      },
+      {
+        "name": "@plone/volto/actions",
+        "message": "Importing from barrel files is not allowed. Please use direct imports of the modules instead."
+      }
+    ],
```

Comment:
@sneridagh this eslint setup for some reason still doesn't work in the editor for me so I could properly try this.

Reply:
@pnicolli they were in the wrong place! Now they are fine.